pkg/apis/local/v1alpha1/localvolumeset_types.go

```diff
@@ -29,6 +29,8 @@ const (
 	Partition DeviceType = "part"
 	// Loop type device
 	Loop DeviceType = "loop"
+	// LVMType is an LVM type
+	LVMType DeviceType = "lvm"
```

Comment:
this also changes auto-provisioning implementation which we shouldn't. cc @rohantmp

Reply:
using `DiscoveredDevicesType` type for LocalVolumeDiscovery so that I doesn't change the auto-provisioning implementation.